packages/hub/node-tests/cards-test.js

```diff
@@ -0,0 +1,241 @@
+
+const JSONAPIFactory = require('../../../tests/stub-searcher/node_modules/@cardstack/test-support/jsonapi-factory');
+const {
+  createDefaultEnvironment,
+  destroyDefaultEnvironment
+} = require('../../../tests/stub-searcher/node_modules/@cardstack/test-support/env');
+
+describe('hub/cards', function () {
+  let env;
+
+  async function teardown() {
+    if (env) {
+      await destroyDefaultEnvironment(env);
+    }
+  }
+
+  describe('card schema', function () {
+    let searchers;
+    //TODO also test: field instructions, field placeholder, field constraints
```

Comment:
I'm still adding these but this should be far enough along to be worth reviewing